azure/Guidelines.md

```diff
@@ -374,6 +375,10 @@ Services, and the clients that access them, may be written in multiple languages
 
 :white_check_mark: **DO** use [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) for date/time.
 
+:white_check_mark: **DO** use a fixed time interval when using durations e.g., `ttlSeconds`. The unit of time should be part of the name, but must at least be documented in the property description.
```

Comment:
Do we have some preference on the units? E.g. preferable second and millisecond?

In extreme case, what if service use [tick](https://docs.microsoft.com/en-us/dotnet/api/system.timespan.ticks?view=net-6.0)?

Reply:
```suggestion
:white_check_mark: **DO** use a fixed time interval when using durations e.g., `ttlSeconds`. The unit of time should be part of the name, e.g. `backupTimeInMinutes` 

✅ **DO** fully document the property and the interval in the description.
```

Any objection to being more direct on the guidance around documentation?
I might also add an example of the operation name.

Reply:
I could give a couple examples like I "e.g., `ttlSeconds` or `backupTimeInMinutes`. Should we just be explicit then that they should *always* include the unit?

@weidongxu-microsoft no, I don't feel we should be explicit about the time unit. It will vary from use to use, and if we made everything milliseconds, many models may need to use a 64-bit integer to at least parse the number where a 32-bit integer would do if in seconds, minutes, hours, etc.

Reply:
Sure. Ask because I've just get caught by spring cloud service use int for cpu/memoryGB, then they decided to support 0.5 cpu and 0.5 GB :-(